internal/primitive/transform/function/strings_functions.go

```diff
@@ -81,3 +81,12 @@ var SplitWithSepFunction = function{
 		return strings.SplitN(s, sep, int(args[2].(float64))), nil
 	},
 }
+var CapitalizeSentence = function{
+	name:      "CAPITALIZE_SENTENCE",
+	fixedArgs: []common.Type{common.String},
+	fn: func(args []interface{}) (interface{}, error) {
+		trimmedSentence := strings.TrimSpace(args[0].(string))
+		capFisrtLetter :=  string(trimmedSentence[0] - 32)
```

Comment:
you can do it only if the first character is between `a` and `z`, but the first character maybe is a capital letter or not a letter

Reply:
if first letter is not letter, do I need to skip capitalization of sentence?

Reply:
or do I need to remove all those special Character? If yes, then we can add new utility function to remove special characters from string.

Reply:
as issue #374  description, I think it only needs to capitalize the first character, if the first character isn't an letter , the function do nothing

Reply:
var CapitalizeSentence = function{
	name:      "CAPITALIZE_SENTENCE",
	fixedArgs: []common.Type{common.String},
	fn: func(args []interface{}) (interface{}, error) {
		value, _ := args[0].(string)
		if len(value) == 0 {
			return value, nil
		}
		if len(value) == 1 {
			return strings.ToUpper(string(value[0])), nil
		}
		if value[0] >= 65 && value[0] <= 90 {
			return value, nil
		} else if (value[0] >= 97) && (value[0] <= 122) {
			return strings.ToUpper(string(value[0])) + value[1:], nil
		} else {
			for i := 0; i < len(value); i++ {
				if (value[i] >= 65 && value[i] <= 90) || (value[i] >= 97 && value[i] <= 122) {
					return strings.ToUpper(string(value[i])) + value[i+1:], nil
				}

			}
		}
		return value, nil
	},
}

I hope This code will work

Reply:
I have tested locally